src/Accelerators/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ if (ACCELERATORS_TO_BUILD)
     add_subdirectory(${t})
     list(APPEND ACCEL_LIST "${t}Accel")
   endforeach(t)
+  message(STATUS "  Accelerators              : ${ACCEL_LIST}")
```

Comment:
Can you make this message(DEBUG) or message(VERBOSE)? The log is already pretty noisy.

Reply:
The noise is mostly produced by third party packages like onnx, not onnx-mlir. I actually would like to see more about what was set and not set, like the summary produced by onnx. The useless lines are the `-- Performing Test ...` lines.